### PR TITLE
Update french translations

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR improve french type translations.

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -16772,6 +16779,7 @@
   [type.noexit]
     en = Dead End
     et = Tupiktee
+    fr = Sans issue
```
Comment (Wivik) [inline data/strings/types_strings.txt:16782]: "Voie sans issue" maybe ?

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -23537,7 +23549,7 @@
     eu = Arteak eta Eskulanak
     fa = ﯽﺘﺳﺩ ﻊﯾﺎﻨﺻ ﻭ ﺮﻨﻫ
     fi = Käsityöt
-    fr = L'artisanat
+    fr = Magasin d'artisanat
```
Comment (Kazing) [inline data/strings/types_strings.txt:23552]:
```suggestion
    fr = Matériel d'arts et de loisirs créatifs
```

Comment (Jean-BaptisteC) [reply data/strings/types_strings.txt]: Fixed

File: data/strings/types_strings.txt (hunk 1 above)
Comment (Jean-BaptisteC) [reply data/strings/types_strings.txt]: Fixed

**Review by Wivik — APPROVED** [review]

**Comment by Jean-BaptisteC** [comment]
@biodranik ready to merge

**Review by biodranik — APPROVED** [review]
Merci !

**Pull request merged**